HTML - 38/memorizeTheOrderOfRedCards.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Red cards</title>
+</head>
+<body>
+<table style="border: 1px solid black; border-spacing: 5px;">

```

Comment:
Add some explanation how to play this. Also when you are clicking cards it's good to mark already clicked cards.

Reply:
- Okay, I can add text above table.
- Will try to mark clicked cards.